### PR TITLE
Kadane's algorithm in python

### LINKED ISSUE #287: Kadane's Algorithm
Write an efficient program to find the sum of contiguous subarray within a one-dimensional array of numbers which has the largest sum.

**Is your feature request related to a problem? Please describe.**
It is not a problem. It is an efficient algorithm to find largest subarray in sum in O(n) complexity

**Describe the solution you'd like**
Simple idea of the Kadane’s algorithm is to look for all positive contiguous segments of the array.

**Additional context**

**Languages**

-   [x] Python : @13snehitha
-   [x] Java : @DefCon-018
-   [x] C: @rgarg-2711
-   [x] C++: @Blackwidow2981

### PR DESCRIPTION
**Pull request by 13snehitha**
# 287 kadane's algorithm full code written in python

Please include a summary of the changes you have made in regard with this issue. 

Fixes #287 

## Type of change

**Please delete options that are not relevant**

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] I have read the contribution guidelines before making this PR.
-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented in my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/kadanes_algo.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+ip=list(map(int,input("Enter array elements").split()))
```
Comment (kaustubhgupta) [inline Python/kadanes_algo.py]: - Poor code formating
- Refactor the whole program with spaces, comments

File: Python/kadanes_algo.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            maxsum=summ
+        i=j
+        while(i<len(ip)-1 and ip[i]<0):
+            i=i+1
+        summ=ip[i]
+        j=i+1
+if(summ>maxsum):
+    maxsum=summ
+print("Maximum positive sum = ",maxsum)
+
```
Comment (kaustubhgupta) [inline Python/kadanes_algo.py]: Remove these extra lines

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- Make changes suggested by code factor
- Update Python folder readme with your entry
- Make these changes:

**Comment by 13snehitha** [comment]
@kaustubhgupta 
Requested changes are updated

**Comment by 13snehitha** [comment]
@kaustubhgupta 
Updated the readme file and program .

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**